routes/job_routes.js

```diff
@@ -44,27 +43,30 @@ router.post('/featuredcubes/rotate', async (req, res) => {
     );
   }
   await Promise.all(notifications);
-  
+
   return res.status(200).send('featured cube rotation check finished successfully');
 });
 
 const tryUpdate = async (podcast) => {
   if (podcast.inactive) {
+    // eslint-disable-next-line no-console
```

Comment:
It might be worth it to just globally disable this rule for the server code

Reply:
Could also extract a wrapper module `logger` for server logs. Then the eslint disable would only need to be in the one module on its different log level methods that wrap the console log. That would also leave a path to later graduate to using a more fully featured logging dependency.

This [blog post describes some good practices for logging with express and nodejs](https://blog.appsignal.com/2021/09/01/best-practices-for-logging-in-nodejs.html). Winston seems like a popular choice and supports structured logging which would be nice for log aggregation and search.

Reply:
Poking around the codebase a bit more, we already have a wrapper than implements this pattern for [cloudwatch](https://github.com/dvndrsn/CubeCobra/blob/fbed9e7fd648a06d902e570129d6a63db8b208a6/serverjs/cloudwatch.js#L103). The methods `cloudwatch.info` and `cloudwatch.error` can be used instead of `console.info`/`console.error` as console log is the fallback when cloudwatch isn't configured.

There's [an example](https://github.com/dvndrsn/CubeCobra/blob/fbed9e7fd648a06d902e570129d6a63db8b208a6/app.js#L142) of using this log wrapper in app.js. Maybe consider replacing console log with a similar pattern?